### PR TITLE
XWIKI-15357: Put a link to explain what to do with an empty XWiki

### PR DESCRIPTION
**Pull request by surli**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm [hunk 1]
```diff
@@ -82,19 +82,8 @@
...
-        #set ($URLartifact = 'xwiki-enterprise-ui-wiki-all')
-      #end
-      #set ($URLversion = $xwiki.getVersion())
-      #if ($URLversion.endsWith('-SNAPSHOT'))
-        #set ($URL = "${URLhost}/snapshots/${URLgroup}/${URLartifact}/$URLversion/")
-      #else
-        #set ($URL = "${URLhost}/releases/${URLgroup}/${URLartifact}/$URLversion/${URLartifact}-${URLversion}.xar")
-      #end
-      #warning($services.localization.render('admin.defaultwikinotinstalled_useflavor'))
+      #set ($URL = "https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/What%20to%20do%20with%20an%20empty%20XWiki/")
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm]: This URL is still too hard to read. Could you rename it to `...AdminGuide/EmptyWiki` for ex?

Comment (vmassol) [reply xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm]: Note that the title of the page can remain :) And be "What to do with an empty wiki?"

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -2239,7 +2239,7 @@ admin.colortheme.wikiSetting=The color theme configured at the wiki level is {0}
 admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
-admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to use the Extension Manager to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
+admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more. <a href="{0}">Click here to find the information how to install it.</a>.
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: I would put the link on "install a flavor". In general I try to never have some text like "click here" when writing HTML. It's a bit unwieldy and doesn't flow too well. WDYT?

File: xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm [hunk 3]
```diff
@@ -82,19 +82,8 @@
...
-        #set ($URLartifact = 'xwiki-enterprise-ui-wiki-all')
-      #end
-      #set ($URLversion = $xwiki.getVersion())
-      #if ($URLversion.endsWith('-SNAPSHOT'))
-        #set ($URL = "${URLhost}/snapshots/${URLgroup}/${URLartifact}/$URLversion/")
-      #else
-        #set ($URL = "${URLhost}/releases/${URLgroup}/${URLartifact}/$URLversion/${URLartifact}-${URLversion}.xar")
-      #end
-      #warning($services.localization.render('admin.defaultwikinotinstalled_useflavor'))
+      #set ($URL = "https://www.xwiki.org/xwiki/bin/view/Documentation/AdminGuide/EmptyWiki/")
```
Comment (vmassol) [inline xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/admin.vm]: You can leave the trailing "/" out.

File: xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties [hunk 4]
```diff
@@ -2239,7 +2239,7 @@ admin.colortheme.wikiSetting=The color theme configured at the wiki level is {0}
 admin.colortheme.manage=Manage color themes\u00BB
 admin.customize=Customize
 admin.save=Save
-admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. You may want to use the Extension Manager to install a flavor, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
+admin.defaultwikinotinstalled_useflavor=Your wiki seems empty. <a href="{0}">You may want to install a flavor</a>, it will bring you a lot of features: user profiles, recent activity, administration pages and many more.
```
Comment (evalica) [inline xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: The link on "install a flavor", not on "you may..."

Comment (evalica) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: I though it was wrong to have HTML syntax in translations.

Comment (evalica) [reply xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties]: text before {0}Link label{1} text after

$services.localization.render(keyid,[param1, param2])
$services.localization.render('admin.defaultwikinotinstalled_useflavor',["<a href='$URL'>", '</a>'])

**Pull request merged**